### DISCUSSION TITLE
DracOS GNU/Linux Remastered

### DISCUSSION THREAD

**Discussion by ekovegeance** [ekovegeance/DracOS_VENOMIZER#4] [Q&A] [answered]
[![Download DracOS Remastered](https://a.fsdn.com/con/app/sf-download-button)](https://sourceforge.net/projects/dracos-remastered/files/latest/download)
## What is Dracos Linux Remastering ? 

DracOS GNU/Linux Remastered ( https://github.com/dracos-linux ) is the Linux operating system from Indonesian , open source is built based on Debian live project under the protection of the GNU General Public License v3.0. This operating system is one variant of Linux distributions, which is used to perform security testing (penetration testing). Dracos linux in Arm by hundreds hydraulic pentest, forensics and reverse engineering. Use a GUI-based tools-tools the software using the CLI (command line interface) and GUI (graphical user interface) to perform its operations. Now Dracos currently already up to version 3.1.5 with the code name "KUNTILANAK WITH REMASTERED".

[![Download DracOS Remastered](https://sourceforge.net/sflogo.php?type=13&group_id=3426069)](https://sourceforge.net/p/dracos-remastered/)

### Teaser 

Link : https://www.youtube.com/watch?v=3x7Mbk7K64g&t=2s 

#### Education 
Dracos Linux is purposed as an educational,especially to recognize the operation system of linux and we respect ethical hacking.

#### Build from source 
had always been built from codes instead of installer,this will stimulate users in indonesia to stay creative and to build the spirit of opensource.

#### Repository
even though proportionally based on codes,Dracos Linux still intends to construct the repository to build up the processes Like Venomizer 

#### Heavy Control
We need to recognize this operating system Very Dificult Because Dracos in build from source code, thus forcing us to compile when installing a package or software, which of course will arise the possibility of system failure and other system vulnerabilities.

#### Always from terminal
None of every singel tool that was installed inside the OS uses GUI. CLI will always consider to particularly openbox to ease the users in need of multi terminal in applying Penetration Testing 

### Penetration Tools List 
- Link : https://github.com/dracos-linux/DracOS_VENOMIZER 
- Information Gathering
- Vulnerability Assessment
- Web Attack
- Exploitation Testing
- Privilege Escalation
- Password Attack
- Social Engineering
- Man In The Middle Attack
- Stress Testing
- Wireless Attack
- Maintaining Access
- Forensics Tools
- Reverse Engineering
- Malware Analysis
- Covering Track

### Link 

- Link : https://github.com/dracos-linux

**Comment by ekovegeance** [comment]
DracOS GNU/Linux Remastered ( https://github.com/dracos-linux ) is the Linux operating system from Indonesian , open source is built based on Debian live project under the protection of the GNU General Public License v3.0. This operating system is one variant of Linux distributions, which is used to perform security testing (penetration testing). Dracos linux in Arm by hundreds hydraulic pentest, forensics and reverse engineering. Use a GUI-based tools-tools the software using the CLI (command line interface) and GUI (graphical user interface) to perform its operations. Now Dracos currently already up to version 3.1.5 with the code name "KUNTILANAK WITH REMASTERED".

**Comment by evsystem7** [comment, marked as answer]
DracOS GNU/Linux Remastered ( https://github.com/dracos-linux ) is the Linux operating system from Indonesian , open source is built based on Debian live project under the protection of the GNU General Public License v3.0. This operating system is one variant of Linux distributions, which is used to perform security testing (penetration testing). Dracos linux in Arm by hundreds hydraulic pentest, forensics and reverse engineering. Use a GUI-based tools-tools the software using the CLI (command line interface) and GUI (graphical user interface) to perform its operations. Now Dracos currently already up to version 3.1.5 with the code name "KUNTILANAK WITH REMASTERED".

**Comment by evsystem7** [comment]
what new in dracos
